pkg/storage/export/export/export.go

```diff
@@ -558,6 +563,8 @@ func (ctrl *VMExportController) manageExporterPod(vmExport *exportv1.VirtualMach
 		}
 	}
 	if pod != nil {
+		// Only create the secret if there any volumes to export. Otherwise it keeps trying over and over
+		// and if the secret already exists it generates errors in the API server 'already exists'
```

Comment:
THis comment seems a little out of context, nowhere in this function are volumes checked

Reply:
Dang forgot to remove this note, yeah it is not needed there let me fix that.